app/queries/drug_stocks_query.rb

```diff
@@ -3,16 +3,17 @@ class DrugStocksQuery
 
   CACHE_VERSION = 1
 
-  def initialize(facilities:, for_end_of_month:)
-    @facilities = facilities
+  def initialize(facility_group:, facilities:, for_end_of_month:, blocks: Region.none)
+    @facilities = Facility.where(id: facilities)
     @for_end_of_month = for_end_of_month
-    # assuming that all facilities on the page have the same protocol
```

Comment:
Instead of assuming, can we raise an error? That'll make sure our current codepaths are solid, and warn devs in the future if they use this thing differently.

Reply:
We have removed this assumption now, we are explicitly passing in a facility group which can only have one protocol.

Reply:
Yup makes sense. This comment is only relevant if we undo the FG and block arguments.